prime-router/src/main/resources/db/migration/V45__fixed_delivery_statistics.sql

```diff
@@ -0,0 +1,105 @@
+/*
+ * The Flyway tool applies this migration to create the database.
+ *
+ * Follow this style guide https://about.gitlab.com/handbook/business-ops/data-team/platform/sql-style-guide/
+ *
+ * Copy a version of this comment into the next migration
+ */
+
+
+/*
+   SPECIAL_DELIVERY_STATISTICS FUNCTION - FIXED.
+  (The active_receiver query had a flaw in it in V44__*.sql - now fixed)
+
+   This answers the question: FOR ALL SENDERS, if a sender sent data for a particular jurisdiction,
+   did it pass filtering, and did we actually send it?
+   
+   This lists all receivers, even those not working, so you have to look at the 'is_actually_receiving' boolean.
+
+   This query cannot distinguish between a failure in the process step, and a filtering action.
+   (Filters are not failures or errors.)
+
+   In PROD this will return dubious results if run with a date less
+   than 24 hours ago, because the data may not have gone to the STLTs
+   yet.
+
+   Example calls.
+   SELECT * FROM special_delivery_statistics('2022-02-04 00:00:00', '2022-02-05 00:00:00');
+   SELECT * FROM special_delivery_statistics((now() - interval '10 minutes')::timestamp, now()::timestamp);
```

Comment:
`special_delivery_statistics` combines all the data over the time interval specified, so there is no sense of "earlier" and "later".

However, it would probably be useful to order the `special_delivery_details` by date, since that lists data report by report.

Reply:
Actually, line 76 of V44 (special_delivery_details) has `RF.created_at > start_time and RF.created_at < end_time`  so if the times are passed in backwards, this would return an empty result.  Sorry to be a pain here.  if start_time > end_time, you could swap them.